includes/rest/class-followers.php

```diff
@@ -115,9 +136,33 @@ public static function request_parameters() {
 			'default' => 1,
 		);
 
+		$params['per_page'] = array(
+			'type' => 'integer',
+			'default' => 10,
```

Comment:
can we use "20" as default, to not break backwards compatibility?

Reply:
Yup seems fine, was just aligning it with the default of 10 from the widget